src/main/webapp/blur-helpers.js

```diff
@@ -93,6 +93,9 @@ function drawImageOnCanvas(image, canvas) {
  * @constructor
  */
 function Rect(rect, image) {
+  // for unblurring feature.
```

Comment:
This comment doesn't really tell much without the context of this PR, i.e. if you look at this code 3 months later you'll have a hard time rememberring what this unblurring feature was. Hence I'd suggest either to expand the comment or to remove it - from the code and the name of the variable it seems quite clear what this variable is for.